python/ray/autoscaler/aws/development-example.yaml

```diff
@@ -57,36 +66,31 @@ file_mounts: {
 }
 
 # List of shell commands to run to initialize the head node.
```

Comment:
are these commands run only on the head node?

Reply:
Oh they are all nodes now. We should update this comment

Reply:
Done